pkgs/by-name/ju/jumper/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  pkgs,
+  lib,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "jumper";
+  version = "0-unstable-2024-08-26";
+  src = fetchFromGitHub {
+    owner = "homerours";
+    repo = "jumper";
+    rev = "264f291754a99ea4aa44d05e6a1a0f4b9159fc8c";
+    hash = "sha256-6++r0mBrV+KhQc7WRiKd8l52CKJUcvzPFqe4S6O6J84=";
+  };
+
+  nativeBuildInputs = [
+    pkgs.which # removes comp warning
+  ];
+
+  buildInputs = [
+    pkgs.fzf # enables fuzzy finding
```

Comment:
Even if `fzf` is put into `buildInputs` it will only be present during the build process.

Even if you used `makeWrapper`'s `wrapProgram` command to add fzf to the `PATH`, it probably wouldn't really work, since fzf is only used in the bash script that's being emitted.

So you probably will have to patch `src/shell.c` to replace all calls to `fzf` by the result of `lib.getExe fzf`.

Reply:
If this sounds difficult, it might be better to put this change into another PR.

Reply:
Ok, good to know. Definitely above my skill level. I'll just remove fzf for now.

Reply:
In that case, `which` can be removed as well, no?

Reply:
Wouldn't be blocking, but maybe something like this would work?

```nix
postPatch = ''
  substituteInPlace src/shell.c \
    --replace-fail "fzf" "${lib.getExe fzf}"
'';
```
We'd really only need to patch [line 180](https://github.com/homerours/jumper/blob/d4edf1438df239fd97c594054337e7f69e594da2/src/shell.c#L180).

Reply:
That is not ideal. This will end up in the user's shell configuration. If fzf gets garbage collected / updated, they would need to update their configuration manually.